eZ Platform/APIs/REST/REST_v3.md

```diff
@@ -17,35 +17,42 @@ As a front-end developer I would like to:
 * Manage the content via the REST API:
     * Use the REST endpoints to quickly create a new object
     * Use the REST endpoints to quickly update an already created object
-    * Use the REST endpoint
 
 # Creating content POST
 The create content endpoint should be simplified to default most values and the simplest possible.
 
 This call should create:
 * A new content object (not a draft) and publish
 * Assign ownership to the current user
-* Create the object in the default language
+* Create the object in the default language if not set
 * Location can be defined either with ID or URL
 
 ## Example: creating a new object
 
-```
-Resource: /content/object
-Method: POST
+__Resource__: _/content/object_
+
+__Accept__: _application/vnd.ez.api.Content+json_
+
+__Content-Type__: _application/vnd.ez.api.SimpleContentCreate+json_
```

Comment:
Ignoring that these are examples and shouldn't be taken literally for implementation for a second:

If we aim for adjusting current current REST api then we should rather use versioning of content type.

@bdunogier What was the format for that again?
`__Content-Type__: _application/vnd.ez.api.ContentCreate+json; version=2.0_` ?

_Version can be something else, point is it''s an alternative version to the current create struct._